packages/server/src/handlers/assets-handler.ts

```diff
@@ -0,0 +1,70 @@
+// import { Handler } from "../lib/handler"
+import { NodeEnv } from "../env/schema"
+import { env } from "../env/server"
+import { Handler } from "../lib/handler";
+import { entryPointPath, getManifest } from "../views";
+
+
+type Assets = {
+    js: string;
+    css: string;
+}
+
+type AssetsLoader = () => Promise<Assets>
+
+// Bun.file is relative to --cwd flag
+const publicPath = (path: string) => `./public/${path}`
+
+// Loading same for both now, but may be usefull later to be able to differentiate
+const loadAssets: AssetsLoader = async () => {
+    const manifest = await getManifest()
+
+    const js = await Bun.file(publicPath(manifest[entryPointPath].file)).text()
+    const css = await Bun.file(publicPath(manifest[entryPointPath].css[0])).text()
+
+    return {
+        js,
+        css,
+    }
+}
+
+const loaders: Record<NodeEnv, AssetsLoader> = {
+    production: loadAssets,
+    development: loadAssets
+}
+
+
+const assets = await loaders[env.NODE_ENV]()
+
+const scriptHandler: Handler = {
+    method: 'GET',
+    path: '/client.js',
```

Comment:
Husk at js og css skal ligge på CDN i prod, og at vi trenger hash i navnet for at det skal funke.

Reply:
Yes, tar det i neste PR :)